.github/workflows/fiddle.yml

```diff
@@ -14,7 +14,6 @@ jobs:
         ruby: [ 2.3, 2.4, 2.5, 2.6, 2.7, head ]
         include:
           - { os: windows , ruby: mingw }
-          - { os: windows , ruby: mswin }
```

Comment:
Could you comment it out with TODO comment to re-add easily in the future?

Reply:
Done.